siva/location.go

```diff
@@ -366,6 +367,7 @@ func (l *Location) Repositories(mode borges.Mode) (borges.RepositoryIterator, er
 			remotes: nil,
 		}, nil
 	}
+	defer repo.Close()
```

Comment:
If the repo is closed here, can we then retrieve the remotes from the config in https://github.com/src-d/go-borges/blob/master/siva/location.go#L374?

Reply:
It's deferred so it's still open when getting the config.